accelerator/apis.c

```diff
@@ -327,7 +335,9 @@ status_t api_send_transfer(const iota_config_t* const tangle, const iota_client_
   flex_trits_to_trytes((tryte_t*)hash_trytes, NUM_TRYTES_HASH, hash243_queue_peek(res->hash), NUM_TRITS_HASH,
                        NUM_TRITS_HASH);
   hash_trytes[NUM_TRYTES_HASH] = '\0';
-  ret = ta_get_transaction_object(service, hash_trytes, txn_obj_res);
+  ta_get_transaction_object_req_push_address(txn_obj_req, hash_trytes);
```

Comment:
When we implemented `send_transfer`, it's only a single zero value transaction. It used transaction hash to get the object. I don't think update `send_transfer` will work in same way.